cmake/CMakeLists.txt

```diff
@@ -1,15 +1,18 @@
 # Minimum CMake required
 cmake_minimum_required(VERSION 2.8.12)
 
-# Project
-project(protobuf C CXX)
-
 # CMake policies
 cmake_policy(SET CMP0022 NEW)
 
+# Project
+project(protobuf C CXX)
+
 # Options
 option(protobuf_VERBOSE "Enable for verbose output" OFF)
+option(protobuf_TARGETS_ALIASES "Provide targets aliases without protobuf:: namespace" ON)
+option(protobuf_MODULE_COMPATIBLE "CMake build-in FindProtobuf.cmake module compatible" ON)
```

Comment:
Ok! We can put the user options in protobuf-options.cmake file and include it in protobuf-config.cmake. Also it can be used in the building of the project, to configure the default behavior.